validation-test/stdlib/Prototypes/PersistentVector.swift.gyb

```diff
@@ -357,10 +357,10 @@ struct _PVSparseVectorNodePointer<Key : Hashable, Value>
     return _valueVectorOffset(layout: layout) + _valueVectorSize(layout: layout)
   }
 
-  var _nodePointer: UnsafeMutablePointer<UInt8>
+  var _nodePointer: UnsafeMutableRawPointer
 
   var _referenceCountPointer: UnsafeMutablePointer<Int> {
-    return UnsafeMutablePointer(_nodePointer)
+    return unsafeBitCast(_nodePointer, to: UnsafeMutablePointer<Int>.self)
```

Comment:
I think we should use `.assumingMemoryBound(to: Int.self)`.  This is a pointer to the reference count, which is always an `Int`.